README.md

```diff
@@ -1,71 +1,188 @@
+## Commands
 
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
-source ~/.bashrc
-nvm install 19.7.0
+### Guild Management
 
-# Install pm2 for process management
-npm install -g pm2
+| Command | Description |
+|---------|-------------|
+| `!invite <username>` | Invite a player to the guild |
+| `!kick <username> [reason]` | Kick a player from the guild |
+| `!promote <username>` | Promote a guild member |
+| `!demote <username>` | Demote a guild member |
+| `!setrank <username> <rank>` | Set a member's rank |
+| `!mute <username> <duration>` | Mute a guild member |
+| `!unmute <username>` | Unmute a guild member |
 
-# Install python3
-sudo apt-get update
-sudo apt install python3-pip
-apt install python3.12-venv
+### Communication Control
 
-# Clone the project
-git clone https://github.com/Jacktheguys/GuildBridgeBot
+| Command | Description |
+|---------|-------------|
+| `!officerchat` | Toggle officer chat mode |
+| `!guildchat` | Toggle regular guild chat mode |
+| `!notifications` | Toggle join/leave notifications |
+| `!toggleaccept` | Toggle auto-accepting guild invites |
 
-# Create directories for bridge bots
-mkdir bridges
+### Miscellaneous
 
-# Create config file
-mv GuildBridgeBot/example.config.json GuildBridgeBot/config.json
-```
+| Command | Description |
+|---------|-------------|
+| `!help` | Display all available commands and bot information |
+| `!online` | Check online guild members |
+| `!list` | Show a list of all guild members |
+| `!override <command>` | Force the bot to use a given command |
 
+Note: Some commands may require appropriate permissions in both Discord and the Hypixel guild.
 
-## Creating A New Bridge Bot
+---
 
-```shell
-# Ensure we are in the home directory
-cd ~
+## Troubleshooting
 
-# Copy files for a new bridge bot
-mkdir bridges/<NAME_OF_GUILD>
-cp -r GuildBridgeBot/* bridges/<NAME_OF_GUILD>/
-cd bridges/<NAME_OF_GUILD>
+1. **Bot Not Starting:** 
+   - Ensure the virtual environment is activated before running the bot. 
+   - Verify that `pm2` is installed globally and the path to the Python interpreter is correct.
 
-# Create a virtal environment
-python3 -m venv venv
-source venv/bin/activate
+2. **Configuration Errors:** 
+   - Double-check the `config.json` file for any misconfigurations or missing values.
 
-# Install requirements
-python3 -m pip install -U -r requirements.txt
+3. **Slow Sync Issues:** 
+   - For message sync delays, review the server and network speed.
+   - Consider enabling Redis for better synchronization.
 
-# Update the config
-nano config.json
+4. **Bot Disconnecting Frequently:**
+   - Check the `config.json` file and ensure auto-reconnection settings are enabled.
+   - Use Redis to prevent disconnections.
 
-# Then run the bot with pm2
-pm2 start main.py --name <NAME_OF_GUILD>-BridgeBot --interpreter ./venv/bin/python --restart-delay=3000
-```
+If you need further assistance, please open an issue on the GitHub repository.
```

Comment:
Can we possibly cut some or most of this out and link to the docs instead? this readme is getting long